src/groovy/au/org/emii/gogoduck/job/Job.groovy

```diff
@@ -37,6 +37,10 @@ class Job {
         new URL("$serverUrl/aggr/$uuid")
     }
 
+    URL getStatusUrl() {
+        new URL("$serverUrl/job/$uuid")
```

Comment:
I'll take this up but you'll have to explain what you mean @danfruehauf as I don't think I'm on the same page and/or URL :grin: